### PR TITLE
Add RPM packaging

### PR DESCRIPTION
**Pull request by mwperina**
- Add new document describing the expected DBUS apis
- Revert accidental rename of "Node" section name in config.
- Complete ListUnit end-to-end handling
- Add RPM packaging

### CODE REVIEW

File: hirte.spec.in [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Name:		hirte
+Version:	@VERSION@
+Release:	@RELEASE@%{?dist}
+Summary:	Container orchestrator for edge based clusters
```
Comment (sdunnagan) [inline hirte.spec.in]: How about, "Hirte is a tool for orchestrating systemd services on remote machines."

File: hirte.spec.in [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+BuildRequires:	gcc
+BuildRequires:	meson
+BuildRequires:	systemd-devel
+
+Requires:	systemd
+
+
+%description
+This package contains the Hirte container orchestrator for edge based clusters.
```
Comment (sdunnagan) [inline hirte.spec.in]: "Hirte is a tool for orchestrating systemd services on remote machines."

File: hirte.spec.in [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+
+%package agent
+Summary:	Hirte container orchestrator agent
+Requires:	systemd
+
+
+%description agent
+This packages contains the agent for Hirte container orchestrator.
```
Comment (sdunnagan) [inline hirte.spec.in]: "This package contains the agent for Hirte."

**Review by sdunnagan — APPROVED** [review]

File: hirte.spec.in (hunk 1 above)
Comment (mwperina) [reply hirte.spec.in]: Just now aligned with suggestions from https://github.com/containers/hirte/pull/97

File: hirte.spec.in (hunk 3 above)
Comment (mwperina) [reply hirte.spec.in]: Just now aligned with comments from https://github.com/containers/hirte/pull/97

**Comment by mwperina** [comment]
@pypingou

File: hirte.spec.in (hunk 2 above)
Comment (mwperina) [reply hirte.spec.in]: Just now aligned with suggestions from https://github.com/containers/hirte/pull/97

File: hirte.spec.in [hunk 4]
```diff
@@ -0,0 +1,72 @@
+Name:		hirte
+Version:	@VERSION@
+Release:	@RELEASE@%{?dist}
+Summary:	Hirte is a container orchestrator tool intended for multi-node low-resource devices
+License:	GPLv2
+URL:		https://github.com/containers/hirte
+Source0:	%{name}-%{version}.tar.gz
+
+BuildArch:	x86_64 aarch64
```
Comment (alexlarsson) [inline hirte.spec.in]: I don't think we should list these only. There is nothing arch-specific about hirte and it should build everywhere if you want.

Comment (mwperina) [reply hirte.spec.in]: Done

**Review by mkemel — APPROVED** [review]

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**